core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/tasks/main.yml

```
@@ -0,0 +1,99 @@
+---
+- name: Install pip3 [Debian]
+  apt:
+    name:
+      - libpq5
+      - python3-psycopg2
+      - python3-jmespath
+    state: present
+    install_recommends: no
+  when: ansible_os_family == 'Debian'
+
+- name: Create postgres_exporter system group
+  group:
+    name: postgres_exporter
+    system: true
+    state: present
+
+- name: Create postgres_exporter system user
+  user:
+    name: postgres_exporter
+    system: true
+    shell: "/usr/sbin/nologin"
+    group: postgres_exporter
+    createhome: false
+
+- name: Set Postgres Exporter file name to install
+  set_fact:
+    exporter_file_name: "postgres_exporter_v{{ exporter.version }}_linux-amd64.tar.gz"
+
+- name: Download Postgres Exporter binaries
+  include_role:
+    name: download
+    tasks_from: download_file
+  vars:
+    file_name: "{{ exporter_file_name }}"
+
+- name: Create /opt/postgres_exporter directory
+  become: yes
+  file:
+    path: /opt/postgres_exporter
+    recurse: yes
+    owner: root
+    group: postgres_exporter
+    mode: 0755
```

We started using human-readable version. :+1:

```suggestion
    mode: u=rwx,go=rx 
```